install_external_source/after/Pods/Pods.xcodeproj

```diff
@@ -116,6 +116,7 @@ Targets:
           PUBLIC_HEADERS_FOLDER_PATH: ''
           SDKROOT: iphoneos
           SKIP_INSTALL: 'YES'
+          SWIFT_ACTIVE_COMPILATION_CONDITIONS: "$(inherited) "
```

Comment:
Yeah, that seems to be the convention, not sure if it's actually required. `"$(inherited) "` also includes a space consistently in the main CocoaPods repo.